### PR TITLE
tools/kothic submdule update to finish #362

### PR DESCRIPTION
**Pull request by Dushistov**
Signed-off-by: Evgeniy A. Dushistov <dushistov@mail.ru>

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,7 +3,7 @@
   url = git://github.com/omapsapp/osmctools.git
 [submodule "tools/kothic"]
   path = tools/kothic
-  url = git://github.com/omapsapp/kothic.git
+	url = git://github.com/organicmaps/kothic
```
Comment (biodranik) [inline .gitmodules:6]: Is it a tab?

Comment (Dushistov) [reply .gitmodules:6]: Yes, this is how `git submodule set-url` works.

Comment (vng) [reply .gitmodules:6]: Hm, but the rest of file uses notation like git://github.com/omapsapp/osmctools.git

@biodranik Should we fix all or leave unchanged?

Comment (biodranik) [reply .gitmodules:6]: I prefer the same delimiters everywhere (spaces). But it's not a blocker, just a nit.

**Review by biodranik — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (vng) [reply .gitmodules:6]: No, I mean that other entries use **omapsapp**/osmctools**.git**

Comment (rtsisyk) [reply .gitmodules:6]: Guys, could you please update .gitmodules in separate PR to 
a) Use https:// instead of git://
b) Use organicmaps instead of maps

Comment (biodranik) [reply .gitmodules:6]: It's better to fix it here.

Comment (rtsisyk) [reply .gitmodules:6]: @Dushistov could you please use the same formatting for .gitmodules? Just to be consistent.

**Comment by rtsisyk** [comment]
I re-based this commit and pushed as 28e9ff10badbb6c8daa60737397113e9772736f1. Thank you for the contribution!

**Pull request closed without merging**